package.json

```diff
@@ -6,6 +6,14 @@
   "scripts": {
     "test": "make test-unit"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/scivey/async-limit.git"
+  },
+  "bugs": {
+    "url": "https://github.com/scivey/async-limit/issues"
+  },
+  "homepage": "https://github.com/scivey/async-limit"
```

Comment:
Missing comma here.  I don't have the fork any more, but should be an easy addition.